projects/widgets/docs/introduction.md

```diff
@@ -1,9 +1,55 @@
-<!-- focus: false -->
+# Widgets
 
-![](../assets/images/logo.png)
+A warm welcome to our Widgets documentation! 👋
 
-## Widgets
+Widgets are ready-made HTML/JS snippets that you can include in your own website to quickly provide a search page for cultural and leisure activities.
 
-A warm welcome to our Widgets documentation! 👋
+publiq provides a widget builder / wizard where you can easily configure your widget. without the need for advanced technical knowledge. During the configuration process all settings are saved on a scripttag. When embedded in a website or app, the scripttag will render the configured agenda with cultural activities.
+
+Widgets can be highly customized and finetuned to your own needs:
+
+* with a query you can define very precisely which events are published in your agenda
+* by simply checking and unchecking checkboxes you can determine which event information you want to show and which you don't
+* there is a broad range of styling options, through CSS or pre-defined themes
+* ...
+
+## Who are widgets (not) for?
+
+### Who are widgets for
+
+Widgets are a very accessible and versatile tool that can appeal to a wide range of partners. Widgets are of particular interest to partners that
```

Comment:
```suggestion
Widgets are a very accessible and versatile tool that can appeal to a wide range of partners. Widgets are of particular interest to partners that:
```

I'm no expert but I think you need to use a colon when continuing a sentence with an unordered list. And you also do it for the list below this one ("We advice to integrate with our Search API instead of widgets if:")